pyoptsparse/pyOpt_error.py

```diff
@@ -23,7 +23,11 @@ def __init__(self, message):
                 i += len(word) + 1
         msg += " " * (79 - i) + "|\n" + "+" + "-" * 78 + "+" + "\n"
         print(msg)
-        Exception.__init__(self)
+        self.message = msg
+        super(Error, self).__init__(message)
```

Comment:
I would prefer the more modern `super().__init__(message)` here

Reply:
Left it in there for clarity, but I agree